components/konflux-rbac/production/base/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 kind: Kustomization
 apiVersion: kustomize.config.k8s.io/v1beta1
 resources:
+  - ../../base
```

Comment:
I'd suggest NOT to share manifests between dev+stage environments and production

```suggestion
```